posts/Jan/2022-04-29-mapping-boolean-types-in-hibernate-orm-6.adoc

```diff
@@ -0,0 +1,149 @@
+= Mapping Boolean attributes in ORM 6
+:awestruct-tags: [ "Hibernate ORM" ]
+:awestruct-layout: blog-post
+:icons: font
+Jan Schatteman <jschatte@redhat.com>
+
+By default, Hibernate maps a Boolean value to a BOOLEAN column. However, not all databases have a built-in BOOLEAN datatype and not all applications will want to use BOOLEAN (legacy schema e.g.).
```

Comment:
Maybe I'm nitpicking, but are we going to use `Hibernate` as synonym for `Hibernate ORM`?

Reply:
Changed it to Hibernate ORM all the same, it's a fair point IMO.